frontend/apps/filemanager/filemanager.lua

```diff
@@ -248,6 +261,28 @@ function FileManager:init()
     self:handleEvent(Event:new("SetDimensions", self.dimen))
 end
 
+function FileManager:fileInformation(file)
+    --local book_stats = {}
+    local file_mode = lfs.attributes(file, "mode")
+    if file_mode == "file" then
```

Comment:
Using return early programming pattern can simplify if else block like this. For example, you can do:

```lua
function FileManager:fileInformation(file)
    if file_mode ~= "file" then return false end
    -- now we are all clear and proceed to do stuff
    local book_stats = DocSettings:open(file):readSetting('stats')
    ...
end
```

This way, we can eliminate many nested if else blocks and improve readability of the code.